mzbench/scenarios/fan_out.bdl

```diff
@@ -0,0 +1,76 @@
+#!benchDL
+
+
+#######
+# Broadcast Scenario:
+# 10k subscribers reading from the same topic "channels/<MF_CHANNEL_ID>/messages"
+# 1 publisher sending 1 msg/s to topic "channels/<MF_CHANNEL_ID>/messages"
+# Overall Msg rate: 1k msg/s
+# Message Size: 100 random bytes
+# Runtime: 5 min
+# QoS level: 2
+# NOTE: Scenario's values can be changed with environment variables predefined in defaults
+# Set environment variables to connect to Mainflux instance:
+# MF_MQTT_ENPOINT = IP/domain of MQTT endpoint on Mainflux
+# MF_MQTT_PORT = Port of MQTT endpoint on Mainflux
+# MF_THING_1_ID = First thing ID of pre-provisioned Mainflux thing
+# MF_THING_1_KEY = First thing key of pre-provisioned Mainflux thing
+# MF_THING_2_ID = Second thing ID of pre-provisioned Mainflux thing
+# MF_THING_2_KEY = Second thing key of pre-provisioned Mainflux thing.
+# MF_CHANNEL_ID = ID of pre-provisioned channel on which both things are connected
```

Comment:
I would prefer `MF_MZBENCH_` prefix

Reply:
Why? To me that looks like unnecessary long name.  This scenario is written in specific language for mzbench and it's in mzbench folder. And this parameters are for Mainflux's endpoint and predefined things, it's not much related to mzbench tool.

Reply:
I proposed this not to be confused with other ENVARS from other services. With this naming we can't know for which purpose it's used. Maybe better `MF_BENCHMARK` or `MF_BENCH`?

Reply:
Maybe then keep rather `MF_MZBENCH`, as user who defines this will have to aware of `MzBench` anyway.

@blokovi this differentiation can help when you see ENV vars in some documentation, etc, to avoid some confusion. This is not necessary, but it is not hard to add, having these long ENV names is not something that would surprise me - especially that this will be mostly copy-pasted from doc.

Reply:
OK, now I understand your point (both of you).

Reply:
I added `MZBENCH_` as prefix to all scenario's variables, not just related to Mainflux connection. So now it's `MZBENCH_MF_CHANNEL_ID` which has more sense  then `MF_MZBENCH_CHANNEL_ID` because it's mainflux channel

Reply:
All env vars now start with `MF_MZBENCH_`